src/systems/ec/implementations/react/templates/pages/research-area-hub/src/ResearchAreaHubPage.jsx

```diff
@@ -0,0 +1,1301 @@
+                      <Link
+                        href="/example"
+                        variant="standalone"
+                        label="Fisheries data collection"
+                      />
+                    </div>
+                    <p className="ecl-u-type-paragraph ecl-u-type-color-black ecl-u-type-color-grey-100 ecl-u-mt-xs ecl-u-mb-none">
+                      Commission (JRC) data collection on fishieries.
+                    </p>
+                  </article>
+                </div>
+              </div>
+              <h2
+                className="ecl-u-type-heading-2 ecl-u-mb-l ecl-u-mt-lg-3xl ecl-u-type-color-black"
+                id="news"
+              >
+                News
+              </h2>
+
+              <article className="ecl-u-border-bottom ecl-u-border-color-grey-15 ecl-u-pv-m ecl-u-mt-m">
+                <div className="ecl-u-type-s ecl-u-type-color-grey-75">
+                  <span className="ecl-u-type-uppercase">Press release</span> |{' '}
+                  <time dateTime="2019-10-17">25 September 2019</time>
+                </div>
+                <div className="ecl-u-type-prolonged-m ecl-u-type-bold ecl-u-mt-xs">
+                  <Link
+                    href="/example"
+                    variant="standalone"
+                    label="Commissioners Miguel Arias, Karmenu Vella and Carlos Moedas welcome the UN report on oceans and climate change"
+                  />
+                </div>
+              </article>
+
+              <article className="ecl-u-border-bottom ecl-u-border-color-grey-15 ecl-u-pv-m">
+                <div className="ecl-u-type-s ecl-u-type-color-grey-75">
+                  <span className="ecl-u-type-uppercase">News</span> |{' '}
+                  <time dateTime="2019-10-17">19 September 2019</time>
+                </div>
+                <div className="ecl-u-type-prolonged-m ecl-u-type-bold ecl-u-mt-xs">
+                  <Link
+                    href="/example"
+                    variant="standalone"
+                    label="Ocean plastic lab returns to Brussels to tackle ocean pollution"
+                  />
+                </div>
+                <p className="ecl-u-type-paragraph ecl-u-type-color-grey-100 ecl-u-mt-xs ecl-u-mb-none">
+                  Back in Brussels from 19 to 30 September 2019, in the
+                  framework of the Research and Innovation Days organised by the
+                  European Commission
+                </p>
+              </article>
+              <Link
+                className="ecl-u-mt-l ecl-u-type-bold"
+                variant="standalone"
+                label="More news"
+                href="/example"
+                icon={{
+                  shape: 'ui--corner-arrow',
+                  size: 'xs',
+                  transform: 'rotate-90',
+                }}
+              />
+
+              <SocialMediaFollow
+                className="ecl-u-mt-l"
```

Comment:
This box becomes full-width in the mobile mockups. Must be taken into account?

Reply:
No, we should stick to the component's display. If this is reported later as an issue, we will update the component, if agreeded by COMM